app/controllers/settings/recruitment/email_templates_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Settings::Recruitment::EmailTemplatesController < AuthorizedController
+  layout "ats/application"
+
+  before_action { authorize! :email_templates }
+  before_action :active_tab
+
+  def index
+    @email_templates_grid = Settings::Recruitment::EmailTemplatesGrid.new(
+      helpers.add_default_sorting(params[:settings_recruitment_email_templates_grid],
+                                  :updated,
+                                  :asc)
+    ) do |scope|
+      scope.page(params[:page])
+    end
```

Comment:
The code partially does not comply with the style guide. Method chaining chapter from [style guide](https://docs.google.com/document/d/1KKYNpomKwr1kv9W5hbjP3fi_8BL_cfKKidL6OIScHcU/edit?tab=t.m7l4vysa5f0i):
```
# Good
variable = var2.method1.method2

# Good
variable =
  var2
  .method1
  .method2
  
# Bad
variable = var2
  .method1
  .method2

# Bad
variable = var2.method1
               .method2

# Bad
variable =
  var2.method1
      .method2
```